hardhat.config.ts

```diff
@@ -17,6 +21,51 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
+task("deployOffsetHelper", "Deploys and verifies OffsetHelper")
+  .addOptionalParam(
+    "verify",
+    "Set false to not verify the OffsetHelper after deployment",
+    true,
+    boolean
+  )
+  .setAction(async (taskArgs, hre) => {
+    const OffsetHelper = await hre.ethers.getContractFactory("OffsetHelper");
+
+    const addressesToUse =
+      hre.network.name == "mumbai" ? mumbaiAddresses : addresses;
+
+    const oh = await OffsetHelper.deploy(tokens, [
+      addressesToUse.bct,
+      addressesToUse.nct,
+      addressesToUse.usdc,
+      addressesToUse.weth,
+      addressesToUse.wmatic,
+    ]);
+    await oh.deployed();
+    console.log(`OffsetHelper deployed on ${hre.network.name} to:`, oh.address);
+
+    if (taskArgs.verify === true) {
+      await oh.deployTransaction.wait(5);
```

Comment:
Can we just remove this? `deployed()` should work for the most part, no?

Reply:
I guess there is still a race between the node we use and the node used by Polygonscan... Is `5` here 5 confirmations?

Reply:
Nope we can't. It takes a while for Polygonscan to index the deployment transaction. Hardhat docs recommend we wait for 5 confirmations in automated tasks to make sure we can actually run the verifications.